go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/target/pod-reaper
+
+go 1.13
+
+require (
+	github.com/docker/distribution v2.7.1+incompatible // indirect
+	github.com/joonix/log v0.0.0-20190130132305-f5f056244ba3
+	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
```

Comment:
Curious about the direct dependencies. Is this actually needed? It seems like a really strange dependency (even being pulled in indirectly)? 

Currently running some testing on my end without it (even though the primary reason is to double/triple check that cron is still behaving as I expect it to)

Reply:
It looks like everything is working without the dependencies commented as `indirect` 
Where those added in by some migration tool?

Reply:
> Where those added in by some migration tool?

i just run `go mod init github.com/target/pod-reaper`. I forgot about `go mod tidy`, made second commit and this left only one indirect dependency. 

> It looks like everything is working without the dependencies commented as indirect
> Is this actually needed? 

what is wrong in knowing what your code is dependent on?

> It seems like a really strange dependency (even being pulled in indirectly)?

this is not a question to me but to maintainer of `sirupsen/logrus` ;)

Reply:
And why it is a problem now but wasn't when the same dependencies were included in `glide.lock` file? go.mod is generated based on information from glide files

Reply:
Looks like Logrus just has it built in support for windows terminal coloring. I guess I shouldn't be too surprised there.

In my head, when I was looking through this, I remember seeing `indirect` in the dependency declaration. Maybe this is some hardcore ingrained PTSD from trying to untangle multiple, conflicting transitive dependencies in huge production applications, but it made me think "oh fun, there was a conflict in the upstream dependencies that needed to be resolved manually". Looking back on it, if one of them didn't include "windows" in it, I probably wouldn't have given it a second thought. I probably would have just assumed it was required to address some deep conflict. The line of code had me jumping at an incorrect conclusion, and it could have just not been there.

> And why it is a problem now but wasn't when the same dependencies were included in glide.lock file? go.mod is generated based on information from glide files

I don't actually know much about go modules, so maybe I should dig in more to ensure I'm not making more assumptions. I figured the `go.sum` file was more akin to the `glide.lock` file and included every hash of every dependency used all the way down. I guess I would have expected the same from the glide.lock file, but not the `glide.yml` file, which I also thought would be akin to the `go.mod` file. 

All in all, this whole thing is me being nitpicky, and I want to assure you that the contribution means a LOT more to me than the stylistic differences in dependency conclusion! I'm almost certainly going to merge this! Odds are you actually know a LOT more about golang than I do!